pages/bookmark/index.js

```diff
@@ -0,0 +1,34 @@
+import { StyledUl } from "@/components/ActivitiesList";
+import { StyledListItem } from "@/components/ActivitiesList";
+import ActivityCardPreview from "@/components/ActivityCardPreview";
+import Link from "next/link";
+import Navigation from "@/components/Navigation";
+
+export default function BookmarkList({ activitiesInfo }) {
+  const filterBookmarks = activitiesInfo.filter(
+    (activityInfo) => activityInfo.isBookmarked === true
+  );
+
+  return (
+    <>
+      <h1>TrailBlazer</h1>
+      {filterBookmarks.length === 0 && (
+        <>
+          <p>You have no bookmarks yet</p>
+          <Link href="/">Go explore all our Activities!</Link>
+        </>
+      )}
+      <StyledUl>
+        {filterBookmarks.map((filterBookmark) => (
+          <StyledListItem key={filterBookmark.id}>
+            <ActivityCardPreview
```

Comment:
After you have organized the state structure in _app.js you should here as well pass the activity as well as the toggleBookmark function. Then toggeling will also work on the bookmark site (it does not yet).